### ISSUE TITLE
Threw  System.IO.IOException: The process cannot access the file 'C:\Users\<username>\AppData\Local\Temp\tmpB23E.tmp' because it is being used by another process

### ISSUE THREAD

**Issue by thangchung** [deislabs/wagi-dotnet#16]
Hi 

I started out following the guidance at the `README` file to create the `wagi-dotnet` project, that was fine and I could build the project successfully. But when I run it, and `curl` as the guidance it always threw the exception as

```
# server.exe
trce: Deislabs.WAGI.Extensions[0]
      Added Route Endpoint for Route: /fibonacci File: modules\fibonacci.wasm Entrypoint: _start
trce: Deislabs.WAGI.Extensions[0]
      Added Route Endpoint for Route: /todos File: modules\get-todos.wasm Entrypoint: _start
info: Microsoft.Hosting.Lifetime[0]
      Now listening on: http://localhost:5000
info: Microsoft.Hosting.Lifetime[0]
      Now listening on: https://localhost:5001
info: Microsoft.Hosting.Lifetime[0]
      Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
      Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
      Content root path: D:\github\rust\webassembly-tour\wagi\server\bin\Release\net5.0
trce: Deislabs.WAGI.Helpers[0]
      Call Module modules\get-todos.wasm Function _start Args  Complete in 00:011 seconds
fail: Microsoft.AspNetCore.Server.Kestrel[13]
      Connection id "0HM96DL0S5J1Q", Request id "0HM96DL0S5J1Q:00000002": An unhandled exception was thrown by the application.
      System.IO.IOException: The process cannot access the file 'C:\Users\thangchung\AppData\Local\Temp\tmpB23E.tmp' because it is being used by another process.
         at System.IO.FileStream.ValidateFileHandle(SafeFileHandle fileHandle)
         at System.IO.FileStream.CreateFileOpenHandle(FileMode mode, FileShare share, FileOptions options)
         at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options)
         at System.IO.StreamReader.ValidateArgsAndOpenPath(String path, Encoding encoding, Int32 bufferSize)
         at System.IO.StreamReader..ctor(String path)
         at Deislabs.WAGI.Helpers.WAGIHost.ProcessRequest()
         at Deislabs.WAGI.Extensions.HttpContextExtensions.RunWAGIRequest(HttpContext context, String wasmFile, String entryPoint, String moduleType, IDictionary`2 volumes)
         at Deislabs.WAGI.Extensions.EndpointRouteBuilderExtensions.<>c__DisplayClass0_0.<<MapWASMModules>b__0>d.MoveNext()
      --- End of stack trace from previous location ---
         at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
         at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application)
``` 

I used Windows 10: 

![image](https://user-images.githubusercontent.com/422341/120618227-152e9180-c485-11eb-9815-e84f16023c39.png)

The reference source code is at https://github.com/thangchung/webassembly-tour/tree/main/wagi/server

**Comment by simongdavies** [comment]
@thangchung Thanks for reporting this, I can repro it, I need to do more investigation, in the meantime can you use WSL2? This should work.

**Comment by simongdavies** [comment]
@thangchung I just released version 0.4.0-preview that includes a fix for this issue.

**Comment by thangchung** [comment]
@simongdavies It worked now. Thank you to let me know
